docs/actions.rst

```diff
@@ -178,6 +181,9 @@ Run a job template.
    * - job_args
      - Additional arguments sent to the job template launch API. Any answers to the survey and other extra vars should be set in nested key extra_vars. Event(s) and fact(s) will be automatically included in extra_vars too. Optionally if the job_args includes an attribute called limit it can be used to over write the limit set from the event payload.
      - No
+   * - lock
```

Comment:
This seems an important feature that many users would use. I feel we should have some more extended documentation with examples in addition to the parameter in the action table.